src/main/java/dev/lambdaurora/lambdynlights/mixin/fabric/AoCalculatorMixin.java

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright © 2023 LambdAurora <email@lambdaurora.dev>
+ *
+ * This file is part of LambDynamicLights.
+ *
+ * Licensed under the MIT license. For more information,
+ * see the LICENSE file.
+ */
+
+package dev.lambdaurora.lambdynlights.mixin.fabric;
+
+import dev.lambdaurora.lambdynlights.LambDynLights;
+import net.minecraft.block.BlockState;
+import net.minecraft.util.math.BlockPos;
+import net.minecraft.world.BlockRenderView;
+import org.spongepowered.asm.mixin.Dynamic;
+import org.spongepowered.asm.mixin.Mixin;
+import org.spongepowered.asm.mixin.Pseudo;
+import org.spongepowered.asm.mixin.injection.At;
+import org.spongepowered.asm.mixin.injection.Inject;
+import org.spongepowered.asm.mixin.injection.callback.CallbackInfoReturnable;
+
+@Pseudo
+@Mixin(targets = "net.fabricmc.fabric.impl.client.indigo.renderer.aocalc.AoCalculator", remap = false)
```

Comment:
I wonder if we can cheese the need for a mixin plugin with the power of optional injections. (This is a thing in mixin but it's not very used)